usr/share/lib/img_proof/tests/SLES/test_sles_motd.py

```diff
@@ -18,6 +18,9 @@ def test_sles_motd(host, get_release_value):
     if 'hardened' in variant:
         pytest.skip('Unable to validate motd in hardened images.')
 
+    if is_sle_micro() and float(version) >= 6.0:
+        pytest.skip('Unable to validate motd in Micro 6.0+ images.')
+
```

Comment:
We should test rather than skip., i.e. in `test_sles_motd` we should allow for the file `/etc/motd` not to exist and if it does not we want to look in `/usr/lib/motd.d/` for the files we put there, there should be at least a `10_header`, a `*doc_links*` and a `welcome` file. If neither `/etc/motd` nor the files in `/usr/lib/motd.d/` exist we fail, otherwise we pass.

@jgleissner is this patter sufficient in your opinion?

Reply:
Thanks for the pointer. Updated the test accordingly.

Reply:
Sounds good to me.